mywebsite/news/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views
+
+
+app_name = 'news'
+urlpatterns = [
+    path('', views.article_list, name='article_list'),
+    path('article/<int:id>/', views.article_detail, name='article_detail')
```

Comment:
Не хватает запятой в конце
```suggestion
    path('article/<int:id>/', views.article_detail, name='article_detail'),
```